crates/cibse_guide_e/src/chapter_6/appendix/common.rs

```diff
@@ -0,0 +1,139 @@
+pub fn area_of_floor(w1: f64, w2: f64) -> f64 {
+    return w1 * w2;
+}
+
+pub fn area_of_floor_equation(a_f: String, w1: String, w2: String) -> String {
+    return format!("{} = {} \\cdot {}", a_f, w1, w2,);
+}
+
+pub fn area_of_opening(wo: f64, ho: f64) -> f64 {
+    return wo * ho;
+}
+
+pub fn area_of_opening_equation(a_o: String, w_o: String, h_o: String) -> String {
+    return format!("{} = {} \\cdot {}", a_o, w_o, h_o,);
+}
+
+pub fn internal_surface_area(a_f: f64, h: f64, w1: f64, w2: f64, a_o: f64) -> f64 {
+    return 2.0 * a_f + 2.0 * h * (w1 + w2) - a_o;
+}
+
+pub fn internal_surface_area_equation(
+    a_net: String,
+    a_f: String,
+    h: String,
+    w1: String,
+    w2: String,
+    a_o: String,
+) -> String {
+    return format!(
+        "{} = 2 \\cdot {} + 2 \\cdot {} ({} + {}) - {}",
+        a_net, a_f, h, w1, w2, a_o
+    );
+}
+
+pub fn ratio_depth_over_height(w1: f64, w2: f64) -> f64 {
+    return w2 / w1;
+}
+
+pub fn ratio_depth_over_height_equation(d: String, w: String, w1: String, w2: String) -> String {
+    return format!("{} / {} = {} / {}", d, w, w2, w1);
+}
+
+pub fn areas_of_openings_multiple_openings(openings_dimensions: Vec<(f64, f64)>) -> Vec<f64> {
+    return openings_dimensions.iter().map(|(w, h)| w * h).collect();
```

Comment:
This is small fish, feel free not to change it, but in a more complex case than this, I would change this to:
```suggestion
    return openings_dimensions.iter().map(|(w, h)| area_of_opening(w, h)).collect();
```